AppCenter/AppCenterTests/MSUtilityTests.m

```diff
@@ -897,6 +897,19 @@ - (void)testObfuscateRedirectUri {
   XCTAssertFalse([obfuscatedString rangeOfString:kMSRedirectUriObfuscatedTemplate].location == NSNotFound);
 }
 
+- (void)testDispatchObjectMacro {
```

Comment:
Please also add a test where `nil` is passed as parameter but is not the last parameter.